tshirts.cpp

```diff
@@ -14,9 +14,9 @@ char size(int cms) {
 }
 
 int main() {
-    assert(size(37) == 'S');
-    assert(size(40) == 'M');
-    assert(size(43) == 'L');
+    assert(size(38) == 'S');

```

Comment:
All boundary values and the values in the range before and after should have test cases.